### PR TITLE
Chroma-Q.Vista3-3.1-21910

### PR DESCRIPTION
**Pull request by JasonCook599**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56207)

### CODE REVIEW

File: manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Chroma-Q.Vista3
+PackageVersion: 3.1-21910
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://www.dropbox.com/s/kmxrsrcj8u2wabd/Vista3_R3_21910.exe?dl=1
```
Comment (Trenly) [inline manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.installer.yaml:9]:
```suggestion
  InstallerUrl: https://www.vistabychromaq.com/register-then-download-the-vista-3-software/?pagename=register-then-download-the-vista-3-software&file=%252FSoftware%2BWindows%2B-%2BVista%2B3%2BRelease%2B3.1%252FVista3_R3_21910.exe
```

File: manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PackageVersion: 3.1-21910
+PackageLocale: en-US
+Publisher: Chroma-Q
+PublisherUrl: https://www.vistabychromaq.com/
+PublisherSupportUrl: https://www.vistabychromaq.com/vista-by-chroma-q-support-request/
+PrivacyUrl: https://www.ac-et.com/Privacy-Policy.pdf
+Author: Chroma-Q
+PackageName: Vista
+PackageUrl: https://www.vistabychromaq.com/register-then-download-the-vista-3-software/
+License: © 1997 - 2022 Chroma-Q
```
Comment (Trenly) [inline manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.locale.en-US.yaml:14]: A copyright is not a license; Please provide a valid license

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Chroma-Q.Vista3
+PackageVersion: 3.1-21910
```
Comment (ItzLevvie) [inline manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.installer.yaml:5]:
```suggestion
PackageVersion: 3.3
```

File: manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Chroma-Q.Vista3
+PackageVersion: 3.1-21910
```
Comment (ItzLevvie) [inline manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.3
```

File: manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Chroma-Q.Vista3
+PackageVersion: 3.1-21910
```
Comment (ItzLevvie) [inline manifests/c/Chroma-Q/Vista3/3.1-21910/Chroma-Q.Vista3.yaml:5]:
```suggestion
PackageVersion: 3.3
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Comment by ItzLevvie** [comment]
You need to update the version folder to 3.3

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**